fluentd/datadog_checks/fluentd/fluentd.py

```diff
@@ -53,29 +64,13 @@ def __init__(self, name, init_config, instances):
     """
 
     def check(self, instance):
```

Comment:
nit:
```suggestion
    def check(self, _):
```

As `instance` should be unused now